pokemongo_bot/cell_workers/snipe_pokemon.py

```diff
@@ -0,0 +1,112 @@
+import json
+import os
+import re
+import sys
+import time
+
+from . import PokemonCatchWorker
+from base_task import BaseTask
+from pokemongo_bot import logger
+from utils import distance
+
+class SnipePokemon(BaseTask):
+
+    def initialize(self):
+        self.api = self.bot.api
+        self.pokemon_list = self.bot.pokemon_list
+        self.position = self.bot.position
+
+    def work(self):
+        if self.bot.tick_count is not 1:
+            return
+
+        while True:
```

Comment:
This makes me super uncomfortable. I understand why it is here and what you are trying to do, but this is super yucky as a worker to have a while true that waits for user input. Perhaps @douglascamata's socket.io approach will be what we need to support this.